src/html/js/common.js

```diff
@@ -166,7 +166,7 @@ Common.setInfo = function(url) {
     Common.accessData.unitUrl = _.first(urlSplit, 3).join("/") + "/";
     Common.accessData.cellUrl = _.first(urlSplit, 4).join("/") + "/";
     Common.accessData.cellName = Common.getCellNameFromUrl(Common.accessData.cellUrl);
-    Common.setBoxUrl(url + "/");
+    Common.setBoxUrl(url);
     Common.accessData.boxName = _.last(urlSplit);

```

Comment:
When "/" is appended to the end, since the last split by "/" becomes empty character, you need to also modify the part of _.last.